### ISSUE TITLE
Defaut publication uses only user._id, but no index is created on that field only

### ISSUE THREAD

**Issue by armellarcier** [Meteor-Community-Packages/meteor-roles#360]
I have slow login method response times (10 seconds for 10 000 role assignments).

Could the `Meteor.roleAssignment.find({ 'user._id': this.userId });` be faster with an index on `user._id` ?

**Comment by StorytellerCZ** [comment]
Yes, that could improve things. @armellarcier are you searching for this on your own or is it when calling a specific function from the package? Looking through the code there seem to be scenarios when using `anyScope: true` might result with a selector with only user ID.
